src/utils/getters.js

```diff
@@ -0,0 +1,37 @@
+import keyBy from 'lodash/keyBy';
+
+export const getId = ({ _id }) => _id.toString();
+
+export const safeGetId = o => o._id || o;
```

Comment:
I can't see where you're using it apart from the tests.

Reply:
hm, it's gone after refactoring :)